lightning/src/chain/keysinterface.rs

```diff
@@ -33,7 +33,7 @@ use util::ser::{Writeable, Writer, Readable};
 
 use ln::chan_utils;
 use ln::chan_utils::{HTLCOutputInCommitment, make_funding_redeemscript, ChannelPublicKeys, LocalCommitmentTransaction, PreCalculatedTxCreationKeys};
-use ln::msgs;
+use ln::msgs::UnsignedChannelAnnouncement;
```

Comment:
Could you open an issue to fix the underlying problem in the bindings generation? Sometimes using the module prefix is preferable.

Reply:
#678